src/TConsole.h

```diff
@@ -191,83 +191,91 @@ class TConsole : public QWidget
 
     TBuffer buffer;
     static const QString cmLuaLineVariable;
-    TTextEdit* mUpperPane;
-    TTextEdit* mLowerPane;
-
-    QToolButton* emergencyStop;
-    QWidget* layer;
-    QWidget* layerCommandLine;
-    QHBoxLayout* layoutLayer2;
-    QWidget* layerEdit;
-    QColor mBgColor;
-    int mButtonState;
-    QColor mCommandBgColor;
-    QColor mCommandFgColor;
-
-    QString mConsoleName;
+    TTextEdit* mUpperPane = nullptr;
+    TTextEdit* mLowerPane = nullptr;
+
+    QToolButton* emergencyStop = nullptr;
+    QWidget* layer = nullptr;
+    QWidget* layerCommandLine = nullptr;
+    QHBoxLayout* layoutLayer2 = nullptr;
+
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
```

Comment:
any reason this is needed here but not previously?

Reply:
Yeah, we support Qt 5.11 minimum and that doesn't have the compile-time `QColorConstants` objects available.

Reply:
Yeah - they didn't come in until - guess what - 5.14 - (or perhaps the old ones started getting depreciated about then - either 5.14 or 5.15 started to get very verbose about deprecated/obsolete stuff - even though for some things they only introduced the *new* way in that same *bloody* version, e.g. [QTBUG-83697](https://bugreports.qt.io/browse/QTBUG-83697)! :frowning:)